README.md

```diff
@@ -203,34 +203,119 @@ All official releases shall be in <code>master</code>. Any updates in between (i
 After a pull request has been open for over 30 days with no activity or response from the author, it'll be automatically marked as stale. We might fork your changes and merge the changes ourselves. Since GitHub tracks contributions by commits, you will be credited.
 </p>
 
-<h2 id="build-yourself">Go Build Yourself</h2>
+<h2 id="building-devicon">Building Devicon</h2>
 <p>
-    Feel free to follow these steps when you want to build the font
-    by yourself.
+Follow these steps to build the website and icons either locally or using <a href=https://www.gitpod.io>Gitpod.io</a>.
 </p>
-<h5>Prerequisites</h5>
-<p>Install <code>gulp</code> (and <code>gulp</code> plugins)</p>
+
+<h3>Table of contents</h3>
+<ol>
+    <li><a href="#using-gitpod">Using Gitpod to build</a></li>
+    <li><a href="#installation">Installation (Linux/Windows)</a></li>
+    <li><a href="#building-icons">Build the icons</a></li>
+    <li><a href="#build-css">Build the CSS stylesheet</a></li>
+    <li><a href="#web-server">Setting up the web server</a></li>
+</ol>
+
+<h3 id="using-gitpod">Using Gitpod.io</h3>
+<p>By using <a href=https://www.gitpod.io)>Gitpod.io</a>, you can easily build the icons and install the<br>required dependencies in one single click. No extra setup is required.</p>
+
+<a href=https://gitpod.io/#https://github.com/devicons/devicon/tree/develop><img src=https://gitpod.io/button/open-in-gitpod.svg alt="Open in Gitpod"></img></a>
+
+<h3 id="installation">Installation</h3>
```

Comment:
I suggest specifying that it's a local installation to better differentiate Gitpod and local installation. Maybe we should also have the other points as subpoints to the local installs, since (if I understand it correctly) they are done automatically when using gitpod.
```suggestion
    <li><a href="#local-installation">Local Installation (Linux/Windows)</a></li>
    <ol>
      <li><a href="#building-icons">Build the icons</a></li>
      <li><a href="#build-css">Build the CSS stylesheet</a></li>
      <li><a href="#web-server">Setting up the web server</a></li>
    </ol>
</ol>

<h3 id="using-gitpod">Using Gitpod.io</h3>
<p>By using <a href=https://www.gitpod.io)>Gitpod.io</a>, you can easily build the icons and install the<br>required dependencies in one single click. No extra setup is required.</p>

<a href=https://gitpod.io/#https://github.com/devicons/devicon/tree/develop><img src=https://gitpod.io/button/open-in-gitpod.svg alt="Open in Gitpod"></img></a>

<h3 id="local-installation">Local Installation</h3>
```